ellama.el

```diff
@@ -492,6 +487,22 @@ CONTEXT contains context for next request."
 
 (defvar ellama--new-session-context nil)
 
+(defun ellama-get-nick-prefix-for-mode ()
+  "Returns preferred header prefix char based om the current mode.
+Defaults to #, but supports org-mode. Depends on ellama-major-mode."
+  (let* ((preferred-mode (symbol-name ellama-major-mode))
+         (prefix-char
+          (cond ((string= preferred-mode "org-mode") ?*)
```

Comment:
it's better to check `(derived-mode-p 'org-mode)` like here https://github.com/s-kostyaev/ellama/blob/90ec5e86bae8c0071f1f44c1ddbb14b62004b71d/ellama.el#L765

Reply:
But this check should be evaluated in session buffer (I think it's already done).